src/NodeServices/Analytics.cs

```diff
@@ -7,6 +7,11 @@ namespace Dynamo.Logging
     /// </summary>
     public static class Analytics
     {
+        /// <summary>
+        /// Disables all analytics collection (Google, ADP, etc.) for the lifetime of the process.
+        /// </summary>
+        public static bool DisableAnalytics;
```

Comment:
Is this using or setting the `AnalyticsUtils.DisableAnalyticsForProcessLifetime` property from Analytics.NET repo?

Reply:
Where is this being set?

Reply:
This is the new public API that will enable integrators to completely shutoff Analytics

Reply:
@pinzart90 is it true that this needs to be used before DynamoModel is constructed to be most effective? Can we add that to the comments?

Reply:
done